Goal App/Services/FirestoreService.swift

```diff
@@ -12,23 +12,19 @@ public class FirestoreService {
     public static let shared = FirestoreService()
     private let database = Firestore.firestore()
 
-    public func create(id: String, task: String, createdDate: Date, dueDate: Date) {
+    func create(id: String, task: String, createdDate: Date, dueDate: Date) {
         let documentReference = database.collection("Goals").document(id)
```

Comment:
I assume `Goals` is an entity name, you can move it to a common struct and use that struct name across the app, the reason for this is suppose you change the entity name in the future then in that case you just need to change the code at one spot
**Example:**
```swift
struct EntityConstant {
static let entityName = "Goals"
}
```